src/method-call-validator/method-call-validator.ts

```diff
@@ -64,28 +64,28 @@ export default class MethodCallValidator {
     params: any,
   ): ParameterValidationError[] | MethodNotFoundError {
     if (methodName === "rpc.discover") { return []; }
-    const method = _.find(this.document.methods, { name: methodName }) as MethodObject;
+    const method = find(this.document.methods, (o: MethodObject) => { return o.name == methodName }) as MethodObject;
 
     if (!method) {
       return new MethodNotFoundError(methodName, this.document, params);
     }
 
-    return _.chain(method.params as ContentDescriptorObject[])
-      .map((param: ContentDescriptorObject, index: number): ParameterValidationError | undefined => {
+    if (method.params) {
+      const paramMap = (method.params as ContentDescriptorObject[]);
+      return compact(paramMap.map((param: ContentDescriptorObject, index: number): ParameterValidationError | undefined => {
         let id: string | number;
-
+        
         if (method.paramStructure === "by-position") {
           id = index;
         } else if (method.paramStructure === "by-name") {
           id = param.name;
         } else {
           if (params[index] !== undefined ) {
             id = index;
-          } else {
+          } else {              
```

Comment:
some formatting issues, i think this repo is missing the `.editorconfig` maybe?

Reply:
added the config